sonar-scanner-engine/src/main/java/org/sonar/scanner/issue/ignore/scanner/IssueExclusionsRegexpScanner.java

```diff
@@ -76,30 +78,32 @@ private void init() {
     currentLineExclusion = null;
   }
 
-  public void scan(String resource, File file, Charset sourcesEncoding) throws IOException {
+  public void scan(String resource, Path filePath, Charset encoding) throws IOException {
```

Comment:
I don't see where `encoding` is used. Previously it was used in `Files.readAllLines`. Is that ok?